openapi.yaml

```diff
@@ -65,6 +61,7 @@ components:
         content_id:
           type: string
           format: uuid
+          nullable: true
```

Comment:
Actually after some digging these can't be null (officially). It's hidden away obtusely [here](https://github.com/alphagov/publishing-api/blob/master/app/queries/get_edition_ids_with_fallbacks.rb#L11)